introduction.md

```diff
@@ -0,0 +1,2 @@
+I am a beginner in a lot of stuff: Web, Java, Javascript with libraries, Ruby, Rails, Perl, Git, C with C++ and C#, Lua, R, Pascal, etc.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.